storage/rocksdb/ha_rocksdb.cc

```diff
@@ -8085,6 +8111,29 @@ int ha_rocksdb::write_row(uchar *const buf) {
   DBUG_RETURN(rv);
 }
 
+rocksdb::Status ha_rocksdb::merge_auto_increment(int curr_id) {
+  Rdb_transaction *const tx = get_or_create_tx(table->in_use);
+
+  auto scf = dict_manager.get_system_cf();
+  uchar key_buf[Rdb_key_def::INDEX_NUMBER_SIZE * 3] = {0};
+  dict_manager.get_key_from_index_id((uchar *)key_buf,
+                                     m_pk_descr->get_gl_index_id());
+  auto k = rocksdb::Slice((char *)key_buf, sizeof(key_buf));
+
+  uchar value_buf[RDB_SIZEOF_AUTO_INCREMENT_VERSION +
```

Comment:
This code looks like the same as Serialize(). Is there way to refactor the code so that they share a common function?